app/js/components/manageApps/ManageApps.jsx

```diff
@@ -362,8 +362,11 @@ export default class ManageApps extends React.Component {
     });
   }
 
-  handleUserClick(name) {
-    toastr.info(`Sorry, there is no open web app for ${name}`);
+  handleUserClick(appType, name) {
+    appType === "module" || appType === "OMOD" ?
```

Comment:
Are you actually testing these changes by running the app? For you should have caught this before my review comment.